azext_iot/tests/__init__.py

```diff
@@ -28,14 +31,23 @@
 USER_ROLE = "IoT Hub Data Contributor"
 DEFAULT_CONTAINER = "devices"
 
+APP_ID = os.environ.get("azext_iot_central_app_id")
```

Comment:
Central test elements should be placed in `tests/central/[__init__].py` rather than the root tests init